src/storm-conv-cli/storm-conv.cpp

```diff
@@ -345,9 +346,8 @@ bool parseOptions(const int argc, const char* argv[]) {
     }
 
     if (general.isVersionSet()) {
-        storm::cli::printVersion("storm-conv");
+        storm::cli::printVersion();
```

Comment:
Why do we no longer need this argument?

Reply:
The name of the binary is from now on only printed through `printHeader` which is the first thing to print for all executions.